admin_ui/src/components/CSVButton.vue

```diff
@@ -59,21 +55,25 @@ export default {
                     )
                     exportedRows.push(...response.data.rows)
                 }
-                return exportedRows
+                let csv: string = "data:text/csv;charset=utf-8,"
+                csv += [
+                    Object.keys(exportedRows[0]).join(";"),
+                    ...exportedRows.map((item) => Object.values(item).join(";"))
+                ]
+                    .join("\n")
+                    .replace(/(^\[)|(\]$)/gm, "")
```

Comment:
What does this regex do?

Reply:
I found it in one of my old PRs (copy from internet) and it's a good question :). Square brackets should be escaped with that regex, but we don't need that at all. I just tried it now and the exported file looks the same without that line. Sorry I didn't think of that before. I will remove that line. Sorry.